app/controllers/api/mentoring/discussions_controller.rb

```diff
@@ -118,7 +118,8 @@ def finish
       render json: {
         discussion: {
           id: discussion.uuid,
-          relationship: SerializeMentorStudentRelationship.(relationship),
+          student: SerializeStudent.(discussion.student, relationship: relationship,
+                                                         anonymous_mode: discussion.anonymous_mode?),
```

Comment:
@kntsoriano Because the student is already part of the initial discussion component, I feel like this is unnecessary now, and could be removed?

Reply:
Sounds good!

Reply:
@kntsoriano Could you do this please? I don't feel confident in the JS to do it on that end.

Reply:
@iHiD I've checked the JS and there seems to be any traces of `relationship`. We should be good.

Reply:
@kntsoriano What I mean is that I feel we don't need to return the `student` here. Which. I think would simplify some of the code in React quite a lot.

Reply:
Oh, I've just checked and I think we can't remove it as the finish endpoint creates the relationship if it does not exist yet.